qlty-plugins/plugins/tests/utils.ts

```diff
@@ -13,6 +13,9 @@ export interface EnvOptions {
 
   /** Test the output against the known good version output of the linter. */
   testAgainstKnownGoodVersion: boolean;
+
+  /** Test the output against the latest snapshots of the linter. */
+  testAgainstLatestSnapshot: boolean;
```

Comment:
The input flags to the testing can get a bit confusing with the various versions. What do you think about adjusting the naming so we have like: `runKnownGoodVersions: boolean` and `compareLatestSnapshots: boolean`. 

So `run*` flags would control what linter version we execute and `compare*` refers to the snapshots

Reply:
We can actually remove testAgainstKnownGoodVersion since it is not being used now that we have compareLatestSnapshots